step-controller/ControllerServer.run.xml

```diff
@@ -3,7 +3,7 @@
     <option name="MAIN_CLASS_NAME" value="step.controller.ControllerServer" />
     <module name="step-controller-backend" />
     <option name="PROGRAM_PARAMETERS" value="-config=&quot;src/test/resources/step.properties&quot;" />
-    <option name="VM_PARAMETERS" value="-Dhttp.keepAlive=true -Dhttp.maxConnections=50 -Dlogback.configurationFile=$PROJECT_DIR$/../step-distribution/step-distribution-parent/step-distribution-controller/template-controller/bin/logback.xml" />
+    <option name="VM_PARAMETERS" value="-Dhttp.keepAlive=true -Dhttp.maxConnections=50 -Dlogback.configurationFile=$PROJECT_DIR$/../step-distribution/step-distribution-controller/template-controller/bin/logback.xml" />
```

Comment:
This requires a specific project layout on the develeper side (i.e., it requires the step-distribution to be checked out in a sibling directory). I'd rather copy the file into this project.

Reply:
This is a fix for the existing configuration currently referring to a path that doesn't exist anymore since the refactoring of the distribution project. It's not the role of this PR to change this behavior.